setup_env.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root" 1>&2
+   exit 1
+fi
+
+if [ "$1" == "-h" ]; then
+    echo "Argument -c allows to install git global name and email."
+    echo "Example setup_env.sh -c username usermail"
+    exit 0
+fi
+
+cp /etc/apt/sources.list /etc/apt/sources.list.backup
+
+. /etc/lsb-release
+
+packages=(
+libssl-dev
+libudev-dev
+libavahi-client-dev
+sqlite3
+libsqlite3-dev
+automake1.11
+libexpat1-dev
+ccache
+libxml2
+libxml2-dev
+git
+libpulse-dev
+subversion
+doxygen
+graphviz
+mscgen
+libbluetooth3
+libbluetooth-dev
+bluez-tools
+chromium-browser
+libgl1-mesa-dev
+libgtest-dev
+binutils
+cmake-curses-gui
+gitk
+)
+
+if [ $DISTRIB_RELEASE!="14.04" ]; then
+   sed -i -e '$a\deb http://ppa.launchpad.net/smspillaz/cmake-2.8.12/ubuntu wily main' /etc/apt/sources.list
+fi
+
+apt-get update
+apt-get install -y ${packages[@]}
+
+#if cmake not installed it will be installed. If any version of cmake installed except 2.8.12 it will be removed before installation.
+if ! dpkg -s cmake | grep "Version" | grep 2.8.12; then
+	apt-get remove -y cmake cmake-data cmake-curses-gui
```

Comment:
@JackLivio I agree that environment in README should be specified version of cmake that SDL required. 

According removing existing packages and installed new versions.
I agree that  it is a bit confusing. But SDL requires specific versions of packages to be installed, and in case if installed versions differs, SDL can fail compiling or even works not correctly. So to create appropriate environment for building and running SDL we need to install appropriate version of dependencies.

I can suggest notify user that setup_env.sh will remove existing version of some packages and install new one. And ask user confirmation according each package (same behavior as yaourt package manager in Archlinux). 
Also I suggest adding ```-f --force``` option to the script, that will scip user confirmation and force installing appropriate versions of dependencies.